### PR TITLE
Markdownify content when using edge and latest shortcodes

### LINKED ISSUE #898: Link to edge docs from installation page is broken and not rendering correctly
**URL of the docs page**

https://docs.radapp.io/installation/

Look at the vscode section

**How is it currently worded?**

image

**How should it be worded?**

It should be a link 😆 

[AB#9871](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/9871)

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.io/community/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
This PR fixes the rendering of edge link on the vscode extension installation section  - https://docs.radapp.io/installation/#step-2-install-the-vs-code-extension

### Summary
:memo::sparkles::wrench:

Updated and added shortcodes for different versions of the Radius Bicep extension docs and improved the installation instructions. The changes aim to enhance the accuracy, readability, and consistency of the docs.

> _`edge` and `latest`_
> _shortcodes use `markdownify`_
> _docs are clearer now_

### Walkthrough
*  Enable Markdown rendering for `edge.html` and `latest.html` shortcodes ([link](https://github.com/radius-project/docs/pull/899/files?diff=unified&w=0#diff-9c9e1cf76b74fb76a3f043ce372c3a44b9010910de605df73e91bda244f73ffdL5-R7), [link](https://github.com/radius-project/docs/pull/899/files?diff=unified&w=0#diff-ee0d505c0030d545c7c9caa3cad3c6b84ee54bdee38c1ada60b0443be0f2255dL5-R6))
*  Use `edge.html` and `latest.html` shortcodes to show different installation instructions for Radius Bicep extension in `install-vscode-bicep.md` ([link](https://github.com/radius-project/docs/pull/899/files?diff=unified&w=0#diff-0a40140e268682122f8739bdadff4f350ddb32e647f44e2858b8fd0ad7b06880L6-R14))
*  Add blank line to the end of `install-vscode-bicep.md` to follow Markdown convention ([link](https://github.com/radius-project/docs/pull/899/files?diff=unified&w=0#diff-0a40140e268682122f8739bdadff4f350ddb32e647f44e2858b8fd0ad7b06880R33))

## Issue reference
Fixes: #898

### CODE REVIEW

File: docs/shared-content/installation/vscode-bicep/install-vscode-bicep.md [hunk 1]
```diff
@@ -30,5 +30,6 @@ To verify you've installed the extension, open any file with the .bicep file ext
 
 5. If running on Windows Subsystem for Linux (WSL), make sure to install the extension in WSL as well:
 
+
```
Comment (Reshrahim) [inline docs/shared-content/installation/vscode-bicep/install-vscode-bicep.md:33]:
```suggestion
   
```

Comment (Reshrahim) [inline docs/shared-content/installation/vscode-bicep/install-vscode-bicep.md]:
```suggestion
```

**Comment by AaronCrawfis** [comment]
@Reshrahim could you target the edge branch with this PR so we can test out the new functionality? Thanks for finding a great solution to this!

**Comment by Reshrahim** [comment]
> @Reshrahim could you target the edge branch with this PR so we can test out the new functionality? Thanks for finding a great solution to this!

this is targeted to edge branch now. Can you take a look?

**Review by AaronCrawfis — APPROVED** [review]
Looks great, thanks!

**Pull request merged**